components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/dto/ResendConfirmationDTO.java

```diff
@@ -42,6 +42,11 @@
      */
     private String successCode;
 
+    /**
+     * Code to used to confirm password recovery when the notifications are managed externally.
```

Comment:
Shall we change the comment as "Code which is used to ...".

Reply:
Fixed be669d5091be5c073ab01d1b036541dc18b37269